spec/models/event_spec.rb

```diff
@@ -1,8 +1,136 @@
 require 'spec_helper'
 
 describe Event do
+  let(:event) { create(:event) }
+  let(:user) { create(:user) }
 
-  describe 'abstract_word_count' do
+  describe 'validations' do
+    it 'has a valid factory' do
+      expect(build(:event)).to be_valid
+    end
+
+    it 'is not valid without a title' do
```

Comment:
Actually you're not testing what your test says and I'd prefer to use [shoulda-matcher](https://github.com/thoughtbot/shoulda-matchers#activemodel-matchers) to test this.